scss/_utilities_text-align.scss

```diff
@@ -10,19 +10,22 @@
 @mixin vf-u-text-align--center {
   .u-text-align--center {
     text-align: center !important;
+    justify-content: center !important;
```

Comment:
Ambiguous rule names within selectors is ok so long as the selector name itself is clear. 

`.u-content-align` sounds like the best approach then.